lib/src/chargebee.dart

```diff
@@ -387,4 +412,24 @@ class Chargebee {
     }
     return NonSubscriptionPurchaseResult.fromJson(jsonDecode(purchaseResult));
   }
+
+  static Map _convertToMap(
+    Product product, {
+    String? customerId = '',
+    CBCustomer? customer,
+  }) {
+    String? id = '';
+    if (customerId?.isNotEmpty == true) {
```

Comment:
Can we simplify it like this?
```suggestion
    if (customerId?.isNotEmpty) {
```